src/YubiKeyAuthListModel.cpp

```diff
@@ -860,7 +860,18 @@ YubiKeyAuthListModel::setAuthData(
                     const uint off = data.bytes[data.size - 1] & 0x0f;
                     const uint miniHash = be32toh(*(guint32*)(data.bytes + off)) & 0x7fffffff;
                     const uint pass = miniHash % maxPass;
-                    const QString strPass(QString().sprintf("%0*u", digits, pass));
+
+                    QString strPass(QString().sprintf("%0*u", digits, pass));
+
+                    if (entry->iName.startsWith("Steam:")) {
```

Comment:
`iName` is an arbitrary display name, not an option 😐

Reply:
I mean, this way any token, not necessarily a Steam token, can be renamed into e.g. "Steam:foo" and that would change the way passwords are generated. That's not right - it should be a separated attribute, the question is where to store it and how to associate it with a particular token. Let me think about it 🤔

Reply:
I did what yubikey manager does and it checks the issuer (since I don't think it's possible to change the method). If it equals "Steam" and the method is TOTP, then it assumes, it's this type of generator. It's just a workaround for bad decision by Valve.
See [this](https://github.com/Yubico/yubikey-manager/blob/b3e69b4450e90a42d200cd9b649a9b45aa284b07/ykman/oath.py#L40-L41) and [this](https://github.com/Yubico/yubikey-manager/blob/88639bc0951b33084ab0a1e6ed55bf666971350e/yubikit/oath.py#L121-L122).
Other otp applications usually treat steam as a separate method, but yubikey couldn't save it.